docs/gestor_dependencias.md

```diff
@@ -0,0 +1,76 @@
+# Gestor de Dependencias
+
+La gestión de dependencias es esencial para asegurar que el proyecto pueda integrarse con bibliotecas externas de manera eficiente, manteniendo su estabilidad y facilidad de mantenimiento. Este documento detalla los gestores de dependencias considerados para el proyecto y las razones detrás de su selección. Se han considerado tres opciones: Apache Maven, Gradle y Apache Ivy.
+
+## Criterios de Selección
+
+Para elegir uno de los gestores, nos basaremos en una serie de criterios:
+
+- *Eficiencia*: Tiempo medio para resolver dependencias en un proyecto estándar (ms).
+
+- *Comunidad*: Cantidad de recursos y paquetes desarrollados, soporte en foros y documentación oficial actualizada.
+
+---
+
+## Gestores de Dependencias Evaluados
+
+### 1. Apache Maven
+
+- **Descripción**: Maven es una herramienta de gestión de proyectos y dependencias ampliamente utilizada en el ecosistema Java. Utiliza un archivo `pom.xml` como base para definir configuraciones y dependencias.
+
+- **Ventajas**:
+  - Compatible con la mayoría de los frameworks y bibliotecas de Java.
+  - Gran cantidad de plugins disponibles para automatizar tareas comunes.
+  - Estructura estándar que facilita la integración con otros equipos y herramientas.
+
+- **Desventajas**:
```

Comment:
Si tenéis una serie de criterios, ¿por qué ponéis luego "ventajas" y "desventajas"? ¿Qué tienen que ver con los criterios?